samples/tooltips/custom-line.html

```diff
@@ -102,9 +102,9 @@
 			tooltipEl.style.opacity = 1;
 			tooltipEl.style.left = positionX + tooltip.caretX + 'px';
 			tooltipEl.style.top = positionY + tooltip.caretY + 'px';
-			tooltipEl.style.fontFamily = tooltip._fontFamily;
-			tooltipEl.style.fontSize = tooltip.fontSize;
-			tooltipEl.style.fontStyle = tooltip._fontStyle;
+			tooltipEl.style.fontFamily = tooltip._bodyFontFamily;
+			tooltipEl.style.fontSize = tooltip.bodyFontSize + 'px';
```

Comment:
Is it `bodyFontSize` or `_bodyFontSize`?

Reply:
Looks like it's `bodyFontSize` but looks weird the other properties are marked private

Reply:
yeah, not entirely sure why that is :|

Reply:
actually, I know .... properties that start with `_` aren't animated so the strings don't need to transition, but other stuff does